command/genesis/predeploy/genesis_predeploy.go

```diff
@@ -0,0 +1,75 @@
+package predeploy
+
+import (
+	"fmt"
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/command/helper"
+	"github.com/spf13/cobra"
+)
+
+func GetCommand() *cobra.Command {
+	genesisPredeployCmd := &cobra.Command{
+		Use:     "predeploy",
+		Short:   "Specifies the contract to be predeployed on chain start",
+		PreRunE: runPreRun,
+		Run:     runCommand,
+	}
+
+	setFlags(genesisPredeployCmd)
+	helper.SetRequiredFlags(genesisPredeployCmd, params.getRequiredFlags())
+
+	return genesisPredeployCmd
+}
+
+func setFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(
+		&params.genesisPath,
+		chainFlag,
+		fmt.Sprintf("./%s", command.DefaultGenesisFileName),
+		"the genesis file to update",
+	)
+
+	cmd.Flags().StringVar(
+		&params.addressRaw,
+		predeployAddressFlag,
+		predeployAddressMin,
+		"the address to predeploy to. Must be >= 0x0000000000000000000000000000000000001100",
```

Comment:
Why not embed `predeployAddressMin` in desctiption?

Reply:
Added:

[37434b6](https://github.com/0xPolygon/polygon-edge/pull/536/commits/37434b616d578c08d800a3eb11fb4a1985fdbe52)